subprojects/docs/src/docs/userguide/publishing_overview.adoc

```diff
@@ -13,78 +13,76 @@
 // limitations under the License.
 
 [[publishing_overview]]
-== Publishing
+= Publishing
 
 The vast majority of software projects build something that aims to be consumed in some way. It could be a library that other software projects use or it could be an application for end users. _Publishing_ is the process by which the thing being built is made available to consumers.
 
 In Gradle, that process looks like this:
 
- 1. Define <<publishing_overview:what,what>> to publish
- 2. Define <<publishing_overview:where,where>> to publish it to
- 3. <<publishing_overview:how,Do>> the publishing
+ 1. Define <<#publishing_overview:what,what>> to publish
+ 2. Define <<#publishing_overview:where,where>> to publish it to
+ 3. <<#publishing_overview:how,Do>> the publishing
```

Comment:
Clicking on these links jumps too far down so the heading is not visible anymore, e.g. after clicking on "What":

<img width="1144" alt="screen shot 2018-07-20 at 12 04 30" src="https://user-images.githubusercontent.com/214207/42996857-1c1b9840-8c15-11e8-9945-0f8ec3a7b832.png">

Reply:
Fixed for all headings.